_episodes/05d-singletons.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Other Data Types"
+teaching: 5
+exercises: 0
+questions:
+- "What are True, False, and None?"
+objectives:
+- "Learn why True, False, and None are different from other datatypes"
+keypoints:
+- "`True`, `False`, and `None` are special keywords in Python"
+- "There is only ever one object called `True`, `False`, or `None`"
+---
+
+# True, False, and None are special data types
+
+ - As their names suggest, use them to store information about whether
+ data is true, false, or does not exist
+ - Exactly one `True`, `False`, and `None` object everytime Python runs
+
+~~~
+a = None
+b = None
+print(id(a), id(b))
```

Comment:
id() seems unnecessary to get this point across. There is no explanation of what id() is or does here and rather just shows up. It either needs explanation or the lesson should explain the concept of None without it.